### PR TITLE
gui support for website specific search

### PR DESCRIPTION
**Pull request by akansh97531**
@nikhilkumarsingh

### CODE REVIEW

**Comment by akansh97531** [comment]
also changed some bugs in gui

**Comment by nikhilkumarsingh** [comment]
@akansh97531  Please wait for #7 to get merged. Then you can add support for same in GUI.

**Comment by akansh97531** [comment]
@nikhilkumarsingh  i added duckduckgo as an option please review

File: ctdl/ctdl.py [hunk 1]
```diff
@@ -31,22 +58,35 @@ def scrape(html):
 	links = []
 	for result in results:
 		link = result.a['href'][7:].split('&')[0]
+		link2 = []
+		link_arr=link.split("/")
```
Comment (nikhilkumarsingh) [inline ctdl/ctdl.py]: Replace this code snippet by just link = link.replace('/blob/', '/raw/')

**Comment by nikhilkumarsingh** [comment]
Please use same function implementations for google and duckduckgo.
Try to avoid extra no. of lines.

**Comment by akansh97531** [comment]
there will be some differences in both implementations
like for query parameter

**Comment by nikhilkumarsingh** [comment]
At least use consistent function names @akansh97531

**Comment by akansh97531** [comment]
renamed function names

**Pull request merged**